app/Models/NetworkEventLog.php

```diff
@@ -177,6 +179,9 @@ public static function create(
             return;
         }
 
+        $banner = Banner::fetchBanner($bannerId);
+        $domain = $banner->campaign->landing_url ?? null;
```

Comment:
I think we should filter domain from landing_url. Note that landing_url could be 1024 chars at most while domain could have 255.

Please consider filtering also in demand controller.

Reply:
Good point!